content/blog/2018/custom-gateway/index.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+publishdate: 2018-10-26
+subtitle: Custom Ingress Gateway
+attribution: Julien SENON
+weight: 89
+keywords: [ingress,traffic-management, aws]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
+
+This post was validated on AWS Infrastructure
+
+## Prerequisites
+
+Have cert-manager installed with [chart](https://github.com/helm/charts/tree/master/stable/cert-manager)
+
+## Cert-Manager Configuration
+
+### Check installation
+
+Cert-manager needs to be installed, if it's has been install with helm you can check installation has bellow:
+
+{{< text bash >}}
+$ helm ls
+NAME         	REVISION	UPDATED                 	STATUS  	CHART                    	APP VERSION 	NAMESPACE
+istio        	1       	Thu Oct 11 13:34:24 2018	DEPLOYED	istio-1.0.2              	1.0.2       	istio-system
+cert        	1       	Wed Oct 24 14:08:36 2018	DEPLOYED	cert-manager-v0.6.0-dev.2	v0.6.0-dev.2	istio-system
+{{< /text >}}
```

Comment:
The entire code-block should be indented. Do not include the output of a command in the same code-block. It makes it confusing for readers to determine what is output and what is a command. Additionally, it makes for a poor experience since readers cannot just copy-paste the command. To fix these issues, I suggest:
{{< text bash >}}
$ helm ls
{{< /text >}}

The output should be similar to the example below and show cert-manager with a `STATUS` of `DEPLOYED`:

{{< text bash >}}
NAME         	REVISION	UPDATED                 	STATUS  	CHART                    	APP VERSION 	NAMESPACE
istio        	1       	Thu Oct 11 13:34:24 2018	DEPLOYED	istio-1.0.2              	1.0.2       	istio-system
cert        	1       	Wed Oct 24 14:08:36 2018	DEPLOYED	cert-manager-v0.6.0-dev.2	v0.6.0-dev.2	istio-system
{{< /text >}}